planning/behavior_path_avoidance_module/include/behavior_path_avoidance_module/helper.hpp

```diff
@@ -343,6 +343,20 @@ class AvoidanceHelper
              parameters_->closest_distance_to_wait_and_see_for_ambiguous_vehicle;
   }
 
+  static bool isAbsolutelyNotAvoidable(const ObjectData & object)
+  {
+    if (object.is_avoidable) {
+      return false;
+    }
+
+    // can avoid it after deceleration.
+    if (object.info == ObjectInfo::NEED_DECELERATION) {
+      return false;
+    }
+
+    return true;
```

Comment:
```suggestion
    // can avoid it after deceleration.
    return object.info != ObjectInfo::NEED_DECELERATION;
```
Depends of Ota-san preference. This is simpler in my opinion.

Reply:
Good suggestion. But I'll add another condition to this function **([see](https://github.com/autowarefoundation/autoware.universe/pull/6898#discussion_r1586936290)).** In that case, I think it's readable if we keep this structure.